templ_attr.cif

```diff
@@ -183,6 +182,70 @@ save_ms_index
     _type.contents               Integer
      save_
 
+save_q_coeff_element
+
+    _definition.update           2024-08-06
+    _description.text                   
+;
+     Element of the matrix _atom_site_Fourier_wave_vector.q_coeff.
+     Notation of the elements is identical to the one used in magCIF.
+     However, in magCIF the number of elements is limited to 3.
+;
+    _type.purpose                Number
+    _type.source                 Assigned
+    _type.container              Single
+    _type.contents               Integer
+    _enumeration.default         0
+save_
+
+save_q_coeff_seq_id
+
+    _definition.update           2024-07-26
+    _description.text                   
+;
+     Element of the matrix _atom_site_Fourier_wave_vector.q_coeff_seq_id.
+     Must match those given in _cell_wave_vector.seq_id.
+;
+    _name.linked_item_id         '_cell_wave_vector.seq_id'
+    _type.purpose                Link
+    _type.source                 Related
+    _type.container              Single
+    _type.contents               Integer
+save_
+
+save_site_ssg_symmetry
+
+    _definition.update           2024-12-12
+    _description.text                   
+;
+      The symmetry code of each atom site as the symmetry operation
+      number 'n' and the higher-dimensional translation 'm1...mp'.
+      These numbers are combined to form the code 'n m1...mp' or
+      n_m1...mp. The character string n_m1...mp is composed as follows:
+      'n' refers to the symmetry operation that is applied to the
+      superspace coordinates. It must match a number given in
+      _space_group_symop_ssg_id. 'm1...mp' refer to the translations
+      that are subsequently applied to the symmetry-transformed
+      coordinates to generate the atom used in calculating the angle.
```

Comment:
The "used in calculating the angle" part here is probably a bit too specific. I assume that this save frame will also be imported by items that do not deal with angles, e.g. _geom_bond.site_ssg_symmetry_1.